pipelines/_ingredients/LSSTComCam/DRP.yaml

```diff
@@ -24,10 +24,8 @@ tasks:
     config:
       connections.goodVisits: goodSeeingVisits
       nVisitsMin: 12
-      # For simulation 2 there are 60 visits per region
-      # between 60399.362 and 60402.419.
-      minMJD: 60399.362
-      maxMJD: 60402.419
+      minMJD: 60399.362  # these need updating
```

Comment:
Whyyyy are these back? For ComCam on sky we're going to be using all the data taken to date to make templates for the next round of observations.

Reply:
I just didn't know if they could just not exist at all, so left them in with a comment to update them, rather than removing something that could have been a required key, as far as I knew. I take it I should just delete these then?

Reply:
Yeah, my preference is to just delete them.

Reply:
Done, and same for LSSTCam.